tests/sentry/integrations/msteams/test_utils.py

```diff
@@ -28,3 +37,103 @@
     "type": "conversationUpdate",
     "id": "f:8e005ef8-f848-156f-55b1-0a5bb3207225",
 }
+
+
+class GetChannelIdTest(TestCase):
```

Comment:
@KKaempen Ok, we should probably not use this file for both constants and an actual spec runner. Can you rename the existing file to something like `test_helpers.py` and let this be an actual spec file?